lametro/management/commands/generate_tag_analytics.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from tqdm import tqdm
+from django.core.management.base import BaseCommand
+
+from lametro.models import LAMetroBill
+
+
+class Command(BaseCommand):
+
+    help = "This command produces a CSV file that lists each Board Report's tags."
+
+    def handle(self, *args, **options):
+
+        with open('tag_analytics.csv', 'w', newline='') as f:
+            writer = csv.writer(f, delimiter=',')
+            files = []
+            for bill in LAMetroBill.all_objects.all():
+                if bill.board_report:
+                    files.append(
+                        (bill.board_report.id,
+                         bill.identifier,
+                         bill.friendly_name,
+                         bill.last_action_date,
+                         bill.subject)
+                    )
+
+            writer.writerow(['File ID', 'Identifier', 'Title', 'Last Action Date', 'Tag'])
```

Comment:
Just had a thought: How about we also include the classification of each tag, so they can look at them in those buckets, as well.

Reply:
(A hint: LAMetroBill instances have a `rich_topics` property that returns a queryset of LAMetroSubject instances corresponding to their subjects.)